### PR TITLE
Ecm 733

### PR DESCRIPTION
**Pull request by gauravchawla73463**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/json/AuthorisationCaseState.json [hunk 1]
```diff
@@ -14,12 +20,12 @@
   {
     "CaseTypeID": "ET_EnglandWales",
     "CaseStateID": "Submitted",
-    "UserRole": "caseworker-employment-api",
-    "CRUD": "CRUD"
+    "UserRole": "caseworker-employment-etjudge-englandwales",
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseState.json:23]: Were these supposed to be changed or was it by accident?

Comment (gauravchawla73463) [reply definitions/json/AuthorisationCaseState.json:23]: It is how GitHub shows that somehow api role has been removed (if you see the next row has been modified as well with api role and submitted state). If you open this branch in IntelliJ, it's there.

  {
    "CaseTypeID": "ET_EnglandWales",
    "CaseStateID": "Submitted",
    "UserRole": "caseworker-employment-api",
    "CRUD": "CRUD"
  },

Comment (hsjhita) [reply definitions/json/AuthorisationCaseState.json:23]: Ah yes, my bad

Comment (hsjhita) [reply definitions/json/AuthorisationCaseState.json:23]: Sorry that's cleared one thing but how about the 'Draft' state?

File: definitions/json/AuthorisationCaseState.json [hunk 2]
```diff
@@ -14,12 +20,12 @@
...
+    "CRUD": "CRU"
   },
   {
     "CaseTypeID": "ET_EnglandWales",
-    "CaseStateID": "Draft",
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseState.json:22]: Draft state removal - see previous conversation

**Review by jthmcts — APPROVED** [review]

**Pull request merged**